sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -378,6 +378,11 @@ object SQLConf {
     .checkValues(Set("hive", "native"))
     .createWithDefault("native")
 
+  val ORC_VECTORIZED_READER_ENABLED = buildConf("spark.sql.orc.vectorizedReader.enabled")
```

Comment:
parquet config: `spark.sql.parquet.enableVectorizedReader`. Shall we have similar config name?

Reply:
Sure, @viirya .